provisioning/templates/nginx-vhost.conf.j2

```diff
@@ -34,7 +34,7 @@ server {
         fastcgi_intercept_errors on;
         fastcgi_read_timeout {{ php_max_execution_time }};
         include fastcgi_params;
-        fastcgi_pass {{ php_fpm_listen }};
+        fastcgi_pass unix:{{ php_fpm_listen }};
```

Comment:
If possible, I'd like to stick with TCP sockets for now... they are slightly more debuggable (in my experience), and the performance impact is negligible in most real-world use cases. Sockets are very slightly faster, but we're talking < 1ms difference in most Drupal usage.